framework/util/platform.h

```diff
@@ -50,6 +50,7 @@
 #include <sys/types.h>
 #include <unistd.h>
 #include <signal.h>
+#include <sys/mman.h>
```

Comment:
I guess the same include from framework/util/page_guard_manager.cpp: 103 can be removed now

Reply:
Yep, removed it